### PR TITLE
[service.partymode.autostart] 0.9.2

### PR DESCRIPTION
**Pull request by elbowz**
### Description
Updated service.partymode.autostart to v0.9.2 to support Kodi 19 matrix requirements.
Above version are available for older Kodi release (eg. Leia).

Checked code with `kodi-addon-checker --branch=matrix --PR`

### Checklist:
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

Thanks!

### CODE REVIEW

File: service.partymode.autostart/PartymodeAutostart.py [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+    def _onScreensaverAction(self):
+        if self.startOnScreensaverPartyMode:
+
+            # if something is in pause and avoidOnPauseStartOnScreensaverPartyMode == true
+            if self.avoidOnPauseStartOnScreensaverPartyMode and xbmc.Player().isPlaying():
+                log('Avoid PartyMode on Screensaver because something is paused')
+
+            else:
+                xbmc.executebuiltin(u'Notification(%s,%s,%s,%s)' % (__addonname__, __language__(30100), 6000, __icon__))
```
Comment (enen92) [inline service.partymode.autostart/PartymodeAutostart.py:126]: xbmcgui.Dialog().notification

File: service.partymode.autostart/addon.xml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.partymode.autostart"
+       name="Partymode autostart"
+       version="0.9.2"
+       provider-name="muttley">
```
Comment (enen92) [inline service.partymode.autostart/addon.xml:5]: Are you the original author? If not add yourself as an author by keeping the credits

File: service.partymode.autostart/addon.xml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    <requires>
+        <import addon="xbmc.python" version="3.0.0"/>
+        <import addon="script.favourites" version="8.1.1"/>
+    </requires>
+    <extension point="xbmc.service" library="PartymodeAutostart.py"/>
+    <extension point="xbmc.python.script" library="main.py">
+        <provides>executable</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en">PartyMode Autostart</summary>
```
Comment (enen92) [inline service.partymode.autostart/addon.xml:15]: en_GB

File: service.partymode.autostart/addon.xml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        <description lang="en">
```
Comment (enen92) [inline service.partymode.autostart/addon.xml:16]: en_GB

File: service.partymode.autostart/addon.xml [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+This service simply executes party mode or favourites on Kodi startup or on screensaver.
+
+Install, enable it (if not already) and enjoy!
+
+If you want change any delays, screensaver and other options go in addon settings.
+        </description>
+        <disclaimer lang="en">The author is not responsible for anything. Please review the code if you do not trust.</disclaimer>
```
Comment (enen92) [inline service.partymode.autostart/addon.xml:23]: en_GB

Comment (enen92) [inline service.partymode.autostart/addon.xml:23]: This disclaimer doesn't make much sense. Aren't you planning on maintaining the add-on?

File: service.partymode.autostart/addon.xml [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+        <license>GNU GENERAL PUBLIC LICENSE Version 3, 29 June 2007</license>
```
Comment (enen92) [inline service.partymode.autostart/addon.xml:24]: GPL-3.0-or-later

File: service.partymode.autostart/main.py [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+    xbmcaddon.Addon().openSettings()
+
+if __name__ == '__main__':
+
+    args = None
+    if len(sys.argv) > 1:
+        args = sys.argv[1:]
+
+    if args:
```
Comment (enen92) [inline service.partymode.autostart/main.py:17]: Unused

File: service.partymode.autostart/main.py [hunk 8]
```diff
@@ -0,0 +1,20 @@
+import xbmc, xbmcaddon
+
+def handleArg(arg):
```
Comment (enen92) [inline service.partymode.autostart/main.py:3]: Unused

File: service.partymode.autostart/addon.xml [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+        <forum>https://forum.kodi.tv/showthread.php?tid=203290</forum>
+        <website>https://github.com/elbowz/partymode-autostart-xbmc-service</website>
```
Comment (enen92) [inline service.partymode.autostart/addon.xml:26]: This info is still correct? If you're taking over an abandoned add-on this should point to the new upstream location of the code

**Review by enen92 — CHANGES_REQUESTED** [review]
See comments. Thanks for the submission

File: service.partymode.autostart/PartymodeAutostart.py (hunk 1 above)
Comment (elbowz) [reply service.partymode.autostart/PartymodeAutostart.py]: Used `xbmcgui.Dialog().notification`

File: service.partymode.autostart/addon.xml (hunk 2 above)
Comment (elbowz) [reply service.partymode.autostart/addon.xml]: Yes, I'm. I use two nickname randomly.
You can check my github profile...anyway I have update this field

File: service.partymode.autostart/addon.xml (hunk 5 above)
Comment (elbowz) [reply service.partymode.autostart/addon.xml]: > This disclaimer doesn't make much sense.

Why?

> Aren't you planning on maintaining the add-on?

Who know...I didn't plan to update this addon...but there was [some users](https://github.com/elbowz/partymode-autostart-xbmc-service/issues/11) request the update and I have discover some free time :)

File: service.partymode.autostart/main.py (hunk 8 above)
Comment (elbowz) [reply service.partymode.autostart/main.py]: Removed...but for me could be stay there. Maybe, could be useful for some new features

File: service.partymode.autostart/addon.xml (hunk 5 above)
Comment (elbowz) [reply service.partymode.autostart/addon.xml]: > This disclaimer doesn't make much sense.

Why?

> Aren't you planning on maintaining the add-on?

Who know...I didn't plan to update this addon...but there was [some users](https://github.com/elbowz/partymode-autostart-xbmc-service/issues/11) request the update and I have discover some free time :)

File: service.partymode.autostart/main.py (hunk 7 above)
Comment (elbowz) [reply service.partymode.autostart/main.py]: Removed...but for me could be stay there. Maybe, could be useful for some new features

File: service.partymode.autostart/addon.xml (hunk 9 above)
Comment (elbowz) [reply service.partymode.autostart/addon.xml:26]: I'm the original developer...

**Comment by elbowz** [comment]
> See comments. Thanks for the submission

Thanks for the deeply and accurate review...make me know if something is still wrong or else I have make some mistake in the PR request/review workflow

**Review by enen92 — APPROVED** [review]

**Pull request merged**